components/notability/commons/notability_checker.lua

```diff
@@ -18,6 +18,8 @@ local _lang = mw.language.new('en')
 local _NOW = os.time()
 local _SECONDS_IN_YEAR = 365.2425 * 86400
 
+NotabilityChecker._EXTRA_LOGGING = true
```

Comment:
So this is a private variable? Why is that?

Reply:
That's a good point, why is it working

Reply:
Ah, Module:Class only privatizes functions, not variables.
I'll drop the `_`.

Reply:
The `_` have been removed.

Reply:
Yes it's just a convention, doesn't actually affect functionality